marklogic-data-hub-central/ui/src/components/explore/entity-type-display-settings-modal/entity-type-display-settings-modal.tsx

```diff
@@ -85,37 +84,35 @@ const EntityTypeDisplaySettingsModal: React.FC<Props> = ({isVisible, toggleModal
     });
   }, [exploreSettingsData, searchText]);
 
-  // ToDo: For use in DHFPROD-8869: Modify Explore Settings to Change hover properties (Structured properties only)
-  // useEffect(() => {
-  //   getEntityModels();
-  // }, []);
-  // const [entityModels, setEntityModels] = useState<any>({});
-  // const getEntityModels = async () => {
-  //   try {
-  //     let response = await axios.get(`/api/models/primaryEntityTypes`);
-  //     if (response.status === 200) {
-  //       let models:any = {};
-  //       response.data.forEach(model => {
-  //         // model has an entityTypeId property, perhaps that should be used instead of entityName?
-  //         models[model.entityName] = model;
-  //       });
-  //       setEntityModels({...models});
-  //     }
-  //   } catch (error) {
-  //     console.error("Error fetching entities", error);
-  //     handleError(error);
-  //   }
-  // };
+  useEffect(() => {
+    getEntityModels();
+  }, []);
+
+  const getEntityModels = async () => {
+    try {
+      let response = await axios.get(`/api/models/primaryEntityTypes`);
```

Comment:
let's refer the "primaryEntityTypes" function (under src/api/queries), for the axios API call here. All API calls should be defined under /api folder only and just referred  in the components.

Reply:
If seems that you basically need the entityModel in this component, so, i think we are already making a call to "/api/models" in the Browse component, Not sure if you get the desired response there but would be good if we can verify it and see if that helps you. We can then avoid this additional call to get the Model.